apps/css-workshop/package.json

```diff
@@ -11,7 +11,12 @@
     "astro": "^4",
     "astro-relative-links": "^0.3.7",
     "backstopjs": "~6.2.1",
-    "npm-run-all": "^4.1.5"
+    "npm-run-all": "^4.1.5",
+    "@astrojs/react": "^4.1.0",
```

Comment:
nit/optional: I notice 4.1.2 was out recently. Should we update the package accordingly as well?